ehcache-impl/src/main/java/org/ehcache/impl/internal/store/heap/SimpleBackend.java

```diff
@@ -28,43 +28,39 @@
 import java.util.Random;
 import java.util.concurrent.atomic.AtomicLong;
 import java.util.function.BiFunction;
-import java.util.function.Supplier;
 
 /**
  * Simple passthrough backend, no key translation
  */
 class SimpleBackend<K, V> implements Backend<K, V> {
 
-  private volatile EvictingConcurrentMap<K, OnHeapValueHolder<V>> realMap;
```

Comment:
This field would stay but become final.

Reply:
Yes, Initialise it at the attribute level and makes it final rather volatile